src/components/components/ebay-notice-base/index.marko

```diff
@@ -53,16 +54,34 @@ $ const {
                     class=iconClass
                 />
             </else-if>
-            <else-if(status === 'education' && type === 'section')>
+            // deprecated status === education. We should probably remove this whole icon on the next major version
+            <else-if(isEducation || (status === 'education' && type === 'section'))>
                 <${educationIcon || LightBulbIcon}
                     a11y-variant='label'
                     a11y-text=a11yIconText || a11yText
                     class=iconClass/>
             </else-if>
+            <if(isEducation && title)>
+                <${title.as || 'h2'}
+                    ...title
+                    class=[`${prefixClass}__title`, title.class]
+                >
+                    <${title}/>
+                </>
+                <if(a11yDismissText)>
+                        <ebay-icon-button
```

Comment:
Since for now Skin isn't using an icon button (because of the display issue with icon button), this should probably be changed to a regular button.

Reply:
Yes. Since skin is applying transparency at the CSS level as long as the icon-btn class is used

Reply:
Sunds good.